analyzers/tests/SonarAnalyzer.UnitTest/TestCases/EmptyNestedBlock.cs

```diff
@@ -90,7 +90,13 @@ public void ConditionalCompilation(object obj)
 #if DEBUG
                 Trace.WriteLine("message");
 #endif
-            } // Noncompliant@-4 FP: the block is not empty
+            }
```

Comment:
coming back, you could also add a test with an enabled processor directive, like at the beginning of the file
```
#define MY_VAL

....

#if MY_VAL
   Trace.Debug();
#endif  
```

and like this you also test when the variable is asserted (or maybe it's redundant?)